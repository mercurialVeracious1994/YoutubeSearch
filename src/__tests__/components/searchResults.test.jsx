import {act, fireEvent, render, screen, waitFor} from '@testing-library/react'
import '@testing-library/jest-dom'
import SearchResults from '../../components/searchResults/searchResults';

describe('searchResults',  () => {
    it("should render searched videos", ()=>{
       const videos = [{
            id:{
                videoId:"videoId"
            },
            snippet:{
                title:"test title",
                description:"test description",
                thumbnails:{
                    high:{
                        url:"test-image-url"
                    }
                }
            }
       }]
       const {container} = render(<SearchResults videos={videos}/>);
       expect(container).toMatchSnapshot();
       
    });

   
})