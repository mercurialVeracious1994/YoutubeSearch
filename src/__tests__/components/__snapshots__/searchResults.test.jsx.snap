// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchResults should render searched videos 1`] = `
<div>
  <section
    class="search-results"
  >
    <section
      class="video"
    >
      <section
        class="video--thumbnail"
      >
        <a
          href="https://www.youtube.com/watch?videoId"
          rel="noreferrer"
          target="_blank"
        >
          <img
            height="200"
            src="test-image-url"
            width="200"
          />
        </a>
      </section>
      <section
        class="video--title-and-description"
      >
        <a
          href="https://www.youtube.com/watch?videoId"
          target="_blank"
        >
          test title
        </a>
        <p>
          test description
        </p>
      </section>
    </section>
  </section>
</div>
`;
